set.mm

```diff
@@ -137379,6 +137384,15 @@ the expression make the whole thing evaluate to zero (on both sides),
       ( vx cr wcel wa cicc co cv w3a cle wbr elicc2 biimp3a simp1d 3expia ssrdv
       ) ADEZBDEZFCABGHZDRSCIZTEZUADEZRSUBJUCAUAKLZUABKLZRSUBUCUDUEJABUAMNOPQ $.
 
+    ${
+      iccssred.1 $e |- ( ph -> A e. RR ) $.
+      iccssred.2 $e |- ( ph -> B e. RR ) $.
+      $( A closed real interval is a set of reals.  (Contributed by Glauco
+         Siliprandi, 11-Dec-2019.) $)
+      iccssred $p |- ( ph -> ( A [,] B ) C_ RR ) $=
+        ( cr wcel cicc co wss iccssre syl2anc ) ABFGCFGBCHIFJDEBCKL $.
+    $}
+
```

Comment:
It seems that this theorem is nested into the block enclosing ~iccssre. This block declares some disjoint variable restrictions which are not necessary for ~iccssred. Therefore, this block for ~iccssred should be moved after the block enclosing ~iccssre, i.e. immediately before ~eliccxr.

Reply:
This seems not to be changed, and also no comment. This theorem must be moved out of the encloing block.

Reply:
I don't understand that one. This is how it is right now in set.mm
```
  ${
    iccssred.1 $e |- ( ph -> A e. RR ) $.
    iccssred.2 $e |- ( ph -> B e. RR ) $.
    $( A closed real interval is a set of reals.  (Contributed by Glauco
       Siliprandi, 11-Dec-2019.) $)
    iccssred $p |- ( ph -> ( A [,] B ) C_ RR ) $=
      ( cr wcel cicc co wss iccssre syl2anc ) ABFGCFGBCHIFJDEBCKL $.
  $}
```

I have copied it outside of the enclosing block immediately above eliccxr
```
  ${
    iccssred.1 $e |- ( ph -> A e. RR ) $.
    iccssred.2 $e |- ( ph -> B e. RR ) $.
    $( A closed real interval is a set of reals.  (Contributed by Glauco
       Siliprandi, 11-Dec-2019.) $)
    iccssred $p |- ( ph -> ( A [,] B ) C_ RR ) $=
      ( cr wcel cicc co wss iccssre syl2anc ) ABFGCFGBCHIFJDEBCKL $.
  $}

  $( A member of a closed interval is an extended real.  (Contributed by Glauco
     Siliprandi, 11-Dec-2019.) $)
  eliccxr $p |- ( A e. ( B [,] C ) -> A e. RR* ) $=
    ( cicc co cxr iccssxr sseli ) BCDEFABCGH $.
```

Please tell me if I've missed something.

Reply:
OK, maybe it was/is the view in GitHub which was/is confusing. If it is as shown in the comment, it is OK.

Reply:
Yeah I get that, I can't even get github to show the diffs. My local editor has also huge troubles with the size of set.mm, I guess git and editors aren't really supposed to handle files with 600k+ lines